main/test/flix/Test.Dec.Enum.Singleton.flix

```diff
@@ -1,4 +1,4 @@
-namespace Test/Dec/OpaqueType {
```

Comment:
File should be moved too.

Reply:
Doh. My bad.

Reply:
I'm not sure I understand. Should the `Test.Dec.Enum.Singleton.flix` file be moved somewhere?

Reply:
It is already correct. I was just stupid.